[WPE] Update Cog version for the WebKit build

clopez opened the pull request:
#### 8b23edce907ac6d759763d409743619947e70ad0
<pre>
[WPE] Update Cog version for the WebKit build
<a href="https://bugs.webkit.org/show_bug.cgi?id=266351">https://bugs.webkit.org/show_bug.cgi?id=266351</a>

Reviewed by Philippe Normand and Michael Catanzaro.

This updates Cog for the WPE build to the last one
(0~0.19.1-1-g843f874)

This fixes an issue with the DRM plugin for the RPi4
that causes Cog to not output anything on the screen.

Cog now depends on libportal, that dependency was added
to the Flatpak SDK in 272033@main

* Tools/PlatformWPE.cmake:

Canonical link: <a href="https://commits.webkit.org/272046@main">https://commits.webkit.org/272046@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d16315089cb7322ce13255e4db8dec5fdcc28e51

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/30467 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/9139 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/32087 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/32977 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/27559 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/31170 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/11376 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/6381 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/27503 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/30775 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/7685 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/27313 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/6582 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/6735 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/27160 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/34316 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/27753 "Passed tests") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/27657 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/32911 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/6720 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/4863 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/30735 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/8466 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/7456 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/3940 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/7263 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

philn commented on Tools/PlatformWPE.cmake:
```diff
@@ -30,7 +30,7 @@ if (ENABLE_COG)
     endif ()
     if ("${WPE_COG_TAG}" STREQUAL "")
-        set(WPE_COG_TAG "9e2dc0a086336d7cd025e50461d126992734672d")
+        set(WPE_COG_TAG "843f874484d09b9e492683a50f4707e584e3fcb6")
```
This in separate PR please.

clopez committed:
[WPE] Update Cog version for the WebKit build
https://bugs.webkit.org/show_bug.cgi?id=266351

Reviewed by Philippe Normand and Michael Catanzaro.

This updates Cog for the WPE build to the last one
(0~0.19.1-1-g843f874)

This fixes an issue with the DRM plugin for the RPi4
that causes Cog to not output anything on the screen.

Cog now depends on libportal, that dependency was added
to the Flatpak SDK in 272033@main

* Tools/PlatformWPE.cmake:

Canonical link: https://commits.webkit.org/272046@main
